### DISCUSSION TITLE
Finetune Models for other Languages

### DISCUSSION THREAD

**Discussion by f-fuchs** [DrewThomasson/ebook2audiobook#141] [Q&A]
Hey,

do you have instructions on how to finetune a model for an other language and what kind of data would you need for that?

Thx
Fabian

**Comment by DrewThomasson** [comment]
Yes, This repo

https://github.com/daswer123/xtts-finetune-webui

## Google colab is broken rn btw

## I you want to rent a GPU easily you can also duplicate this hugginface space I made

https://huggingface.co/spaces/drewThomasson/xtts-finetune-webui-gpu

## A space you can use to de-noise the training data also

https://huggingface.co/spaces/drewThomasson/DeepFilterNet2_no_limit

**Comment by DrewThomasson** [comment]
I'll see if I can add that to the readme cause a lot of people are asking about that
